environment.yml

```diff
@@ -22,7 +22,6 @@ dependencies:
   - python-dateutil>=2.1
   - setuptools
   - setuptools_scm
-  - wxpython
```

Comment:
I'm unclear whether removal is ok. The original purpose of `environment.yml` is to provide contributors with an easy way to set up a dev environment. We should how complete that should be in terms of optional dependencies, but I'm inclined to make a dev environment as complete as possible.